### PR TITLE
Add yarn/berry switch in create-turbo tests.

### PR DESCRIPTION
**Pull request by nathanhammond**
This would be somewhat cleaned up in a `corepack` world.

### CODE REVIEW

File: packages/create-turbo/__tests__/cli.test.ts [hunk 1]
```diff
@@ -325,10 +331,27 @@ function cleanupTestDir() {
   if (fs.existsSync(testDir)) {
     fs.rmSync(testDir, { recursive: true });
   }
+  fs.rmSync(path.join(__dirname, "../.yarn"), { recursive: true, force: true });
+  fs.rmSync(path.join(__dirname, "../.yarnrc"), { force: true });
+  fs.rmSync(path.join(__dirname, "../.yarnrc.yml"), { force: true });
```
Comment (mehulkar) [inline packages/create-turbo/__tests__/cli.test.ts:336]: Are these getting created because of `yarn set version`?

**Review by mehulkar — APPROVED** [review]
I think this makes sense, thanks 👍🏾. Getting parallels set up for myself also

**Comment by nathanhammond** [comment]
oh no, this is in a broken state! Let me get a new PR up...

**Comment by nathanhammond** [comment]
(sorry, didn't expect this to get merged as I kept pulling and pulling on the thread)

**Pull request merged**